keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -266,17 +281,24 @@ _This section must be completed when targeting beta graduation to a release._
 * **How can a rollout fail? Can it impact already running workloads?**
   Try to be as paranoid as possible - e.g., what if some components will restart
    mid-rollout?
+  The rollout should not fail. Feature gate only needs to be enabled on `kube-controller-manager`. So it is either enabled or disabled.
```

Comment:
In an HA cluster, it's possible to have a 1.N kube-controller-manager take the lease, then lose it to a 1.N-1 kube-controller-manager.  What happens in that case?

Reply:
Added what will happen in this case in the KEP.